docs/en/latest/practices/how-to-use-go-plugin-runner-in-apisix-ingress.md

```diff
@@ -0,0 +1,205 @@
+---
+title: How to use go-plugin-runner with APISIX Ingress
+---
```

Comment:
Include keywords and description here for SEO.

Reply:
What exactly should I do? Can you give a demonstration case?

Reply:
ping @navendu-pottekkat

Reply:
ping @navendu-pottekkat

Reply:
@Chever-John Could you check this? https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/api-breaker.md

See the YAML front matter there.

Reply:
Thanks, that helps me~

Reply:
Solved~